docker/ts_config/timesketch.conf

```diff
@@ -0,0 +1,12 @@
+DEBUG = False
+SECRET_KEY = u'dockerdockerdocker'
```

Comment:
For now, just remove dockerdockerdocker with nothing, so you manually have to put in a secret.